app/components/goodcity/add-item-overlay.js

```diff
@@ -29,44 +28,72 @@ export default Ember.Component.extend(AsyncMixin, {
   },
 
   totalNumberTomove: Ember.computed(
-    "pkg.packagesLocations.@each.defaultQuantity",
+    "pkg.packagesLocations.@each.defaultAddableQuantity",
     function() {
-      return this.calculateSumFor("defaultQuantity");
+      return this.calculateSumFor("defaultAddableQuantity");
     }
   ),
 
-  addRemoveItem(params) {
-    this.runTask(
-      this.get("packageService")
-        .addRemoveItem(this.get("entity.id"), params)
-        .then(this.set("open", false))
-        .catch(response => {
-          this.get("messageBox").alert(
-            response.responseText.errors[0] ||
-              this.get("i18n").t("unexpected_error")
-          );
-        })
-    );
+  resolvePromise() {
+    let promises = this.createPromises();
```

Comment:
`createPromises` seems generic name. But this method name is not generic it is returning array of promises for Removing of Item.

Reply:
sure 👍